test/integration/client-side-encryption/driver.test.ts

```diff
@@ -50,6 +50,67 @@ describe('Client Side Encryption Functional', function () {
   const keyVaultCollName = 'datakeys';
   const keyVaultNamespace = `${keyVaultDbName}.${keyVaultCollName}`;
 
+  describe('ClientEncryption', metadata, function () {
```

Comment:
These seem to duplicate the prose tests in [client_side_encryption.prose.26.custom_aws_credential_providers.test.ts](https://github.com/mongodb/node-mongodb-native/pull/4373/files#diff-50fba3dd7d938f39c784503d256c238fbea0061d70fe30dce55ee406d336870c)?

Reply:
These two are now removed.